Wrote hackathon logistics Section (#9)
kdesai2018/ultimate-hackathon-starting-guide: hackathon-logistics/README.md

A hackathon is an invention marathon, where people come together, using their technical and creative skills to create solutions for everyday problems. It's a space where people jump off each others energy and create something out of nothing but a spark of an idea. What I like the most about hackathons is just the energy of innovation, knowing that everyone there is  making something they are passionate about and are excited about.
      
#### What should I do before I come to the event?
There's very little you have to do before the actual event--most of the magic happens there. Just be sure to look over the hackathon's pre-event email and follow whatever instructions they have for you. I would also make sure you are well rested the night before and ready to create the next day. If you would like, I would recommend finding a team before the event starts, whether that's friends that you registered with or people that you have connected with online. It's also good to brainstorm some ideas (more on this linked [here](https://github.com/kdesai2018/ultimate-hackathon-starting-guide/blob/master/ideation/README.md)) and come in with a project idea. 

#### What should I expect?

> Most hackathons are held in a big room of some sort with tables for teams to be at. There will typically be meals provided, lots of snacks at all times, and events throughout. At HackTX we had events like a CTF, workshops, and a paper airplane making contest. Definitely check these out--they're lots of fun and you can meet some really cool people. Talk to your peers and mentors--we are here to help! There are so many resources availible to you at a hackathon: companies to talk to (and swag to get;), mentors, organizors, and more. Most of the time people will be working and some people will event stay up through the entire night. Although this can be part of the hackathon experience, don't feel like you have to do this. Sleep is important to productivity too. At the end, most hackathons will have teams demo to judges and give the top teams prizes. I would recommend demoing regardless of how your project turns out. It's a good experience just to talk through your idea, feel proud of what you made, and explain the process to a judge. Most of all, you should expect to have fun! Expect to learn, meet new friends, and enjoy a great environment and driven community. 

Reviewer: ```suggestion
Most hackathons are held in a big room of some sort with tables for teams to be at. There will typically be meals provided, lots of snacks at all times, and events throughout. At HackTX we had events like a CTF ([Capture The Flag](https://ctfd.io/whats-a-ctf/)), workshops, and a paper airplane making contest. Definitely check these out--they're lots of fun and you can meet some really cool people. Talk to your peers and mentors--we are here to help! There are so many resources available to you at a hackathon: companies to talk to (and swag to get), mentors, organizers, and more. Most of the time people will be working and some people will even stay up the entire night. Although this can be part of the hackathon experience, don't feel like you have to do this. **Sleep is important for productivity too**. At the end, most hackathons will have teams demo to judges and give the top teams prizes. I would recommend demoing regardless of how your project turns out. It's a good experience just to talk through your idea, feel proud of what you made, and explain the process to a judge. Most of all, you should expect to have fun! Expect to learn, meet new friends, and enjoy a great environment and driven community. 
```

#### What should I bring?

> - Yourself

Reviewer: ```suggestion
- Yourself
- A government or school-issued ID card
```
```suggestion
- Yourself
```

- Your laptop and chargers
- Any other equipment you deem necessary for your project (i.e. a drawing tablet)
- A blanket and/or sweatshirt (it might be cold)
- Most of all, bring energy, ideas, and a great attitude so you can have tons of fun!

> 

Reviewer: ```suggestion
Sometimes a hackathon will have a "hardware lab", where you can check out components like Arduinos, VR headsets and more! Check with the hackathon organizers for more details
```

#### What can I work on?
You can work on anything that uses technology! This can be a website/app, a video game, a VR/AR app, etc. Use your imagination to come up with a unique tech project! 

#### Where do I start?

> It can be very overwhelming to go into a hackathon for the first time. With so many people everywhere and things going on, it's hard to figure out where to begin and what to do with your time. Here are my suggestions. First, form a team and brainstorm a project idea. Then, figure out how you are going to create that project. Think about what technologies y'all will need, figure out a good delegation of who should do what, break down the project into achievable steps and focus on create a minimal viable product (MVP). After this, your team can start hacking! Most of the time this means digging through the internet to learn more about the tech you need or finding examples that can help you achieve your goals.

Reviewer: ```suggestion
It can be very overwhelming to go into a hackathon for the first time. With so many people everywhere and things going on, it's hard to figure out where to begin and what to do with your time. Here are my suggestions. First, form a team and brainstorm a project idea. Then, figure out how you are going to create that project. Think about what technologies y'all will need, figure out a good delegation of who should do what, break down the project into achievable steps and focus on create a minimal viable product (MVP). After this, your team can start hacking! Most of the time this means digging through the internet to learn more about the tech you need or finding examples that can help you achieve your goals.

Check out the ideation page in this guide for a more detailed insight into the process. 
```

- Yourself
- A government or school-issued ID card
- Your laptop and chargers
- Any other equipment you deem necessary for your project (i.e. a drawing tablet)*
- A blanket and/or sweatshirt (it might be cold)
- Most of all, bring energy, ideas, and a great attitude so you can have tons of fun!

> *Keep in mind that sometimes a hackathon will have a "hardware lab", where you can check out components like Arduinos, VR headsets and more! Check with the hackathon organizers for more details

Reviewer: ```suggestion
* Keep in mind that sometimes a hackathon will have a "hardware lab", where you can check out components like Arduinos, VR headsets and more! Check with the hackathon organizers for more details 
```
